notebooks/pyproject.toml

```diff
@@ -10,6 +10,7 @@ ipykernel = "^5.5.3"
 pandas = "^1.2.4"
 plotly = "^4.14.3"
 nbstripout = "^0.3.9"
+sklearn = "^0.0"
```

Comment:
https://pypi.org/project/sklearn/ says to use `scikit-learn` instead.

vscode also decided that it wanted to install `notebook` when I tried things out on a fresh virtualenv, but I can make a patch for that as a separate PR.